components/Button.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export default function Button({ children, className, onClickEvent }) {
+  return (
+    <StyledButton className={className} onClick={onClickEvent}>
```

Comment:
The same here for the prop `className` as mentioned above. I do not understand why you need this prop. 🤔